tests/unit/handlers/helloCommand.test.ts

```diff
@@ -0,0 +1,21 @@
+import { InteractionResponseType } from "discord-interactions";
+import { helloCommand } from "../../../src/controllers/helloCommand";
+import { responseJson } from "../../../src/typeDefinitions/default.types";
+import JSONResponse from "../../../src/utils/JsonResponse";
+
+describe("Test helloCommand function", () => {
```

Comment:
we don't need `Test` in the quotes here, when using `describe` block, just highlight the components/function which is being tested. In this case "helloCommand function" is fine. Same goes for other tests.

Reply:
can you give me a reference to any docs that say this

Reply:
This is a general convention, you can verify by checking every `describe` usage in https://jestjs.io/docs/getting-started.
Writing `Test` would be unnecessary since the only useful information we have here is the component which is being tested.